types/types.go

```diff
@@ -432,6 +432,21 @@ func ValidateReplicaCount(count int) error {
 	return nil
 }
 
+func ValidateReplicaAutoBalance(option ReplicaAutoBalance) error {
+	switch option {
+	case ReplicaAutoBalanceIgnored:
+		return nil
+	case ReplicaAutoBalanceDisabled:
+		return nil
+	case ReplicaAutoBalanceLeastEffort:
+		return nil
+	case ReplicaAutoBalanceBestEffort:
+		return nil
```

Comment:
nit: You can use a multi value case
```suggestion
	case ReplicaAutoBalanceIgnored, ReplicaAutoBalanceDisabled, ReplicaAutoBalanceLeastEffort, ReplicaAutoBalanceBestEffort:
		return nil
```

Reply:
Thanks, fixed.